### PR TITLE
allow aditional serviceAccount annotations

### PR DESCRIPTION
**Pull request by cesarokuti**
Signed-off-by: Cesar Okuti <cesar.okuti@gmail.com>

#### What this PR does / why we need it:
I need to configure service account annotations for internal security configurations.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [X] Chart Version bumped
- [X] `CHANGELOG.md` has been updated
- [X] Variables are documented in the `README.md`

### CODE REVIEW

**Review by lucasfuzikawa — APPROVED** [review]

**Comment by CharlyF** [comment]
Thanks for the contribution - I see that we have this in other charts, this looks good to me. 
Can you just rebase? I'll approve thereafter.

**Review by kayayarai — APPROVED** [review]
Docs updates look good! thanks!

**Comment by cesarokuti** [comment]
> Thanks for the contribution - I see that we have this in other charts, this looks good to me. Can you just rebase? I'll approve thereafter.

@CharlyF can you rerun the tests to merge it.

File: charts/datadog-operator/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.7.11
+
+* Allows aditional service account annotations.
```
Comment (L3n41c) [inline charts/datadog-operator/CHANGELOG.md:5]:
```suggestion
* Allow additional service account annotations.
```

**Pull request merged**